tasks/os-check.yml

```diff
@@ -3,3 +3,18 @@
   fail:
     msg: "The Datadog Ansible role does not support your OS yet. Please email support@datadoghq.com to open a feature request."
   when: ansible_facts.os_family not in ["RedHat", "Rocky", "AlmaLinux", "Debian", "Suse", "Windows", "Darwin"]
+
+- name: Upper bound last supported Agent version for Centos < 7
+  when: ansible_facts.os_family == "RedHat"
+  block:
+    - name: Get RHEL major version equivalent
+      command: "rpm -E %{rhel}"  # noqa: command-instead-of-module
```

Comment:
Hmm, this is interesting, I never thought about doing this. Have you tested on e.g. Scientific Linux 6? I feel like there might be rhel-6 based distros where this macro wouldn't expand, but I'm not sure.

Reply:
I did a quick test on a Scientific Linux 6 box and it did expand as expected (to 6). Would you say it's safer to use something else (or at least to have some sort of fallback)?